components/DetailPageSidebar/DetailPageSidebar.js

```diff
@@ -0,0 +1,166 @@
+                    )}
+
+                    <p className="details-description">{artifact.description}</p>
+
+                    { artifact.tag !== 'ibm_experimental' &&
+                        <div className="artifact-margin-addition"></div>
+                    }
+                    { artifact.tag !== 'ibm_experimental' &&
+                        <div className="tag-container">
+                            { isStandardAccount && artifact.accountTag &&
+                                <div className="account-tag">{artifact.accountTagName}</div>
+                            }
+                            <div className="provider-tag">{artifact.tagName}</div>
+                            { artifact.availabilityTag && artifact.availabilityTag !== 'public' &&
+                                <div className="availability-tag">{artifact.availabilityTagName}</div>
+                            }
+                            { artifact.stageTag &&
+                                <div className="stage-tag">{artifact.stageTagName}</div>
+                            }
+                        </div>
+                    }
+                    {children}
+                    <div className="description-tag-container">
+                        { artifact.docURL &&
+                            <a className="description-tag-docs-link" href={artifact.docURL} target="_new">{i18n.viewDocs}</a>
+                        }
+                        { artifact.type === 'softlayer' && artifact.termsUrl &&
+                            <a className="description-tag-terms-link" href={artifact.termsUrl} target="_new">{i18n.viewTerms}</a>
+                        }
+                    </div>
+                    <div className="details-container">
+                        { 
+                            this.generateBlocks([
+                                { type: 'author', cssName: 'author', label: i18n.author, value: artifact.author },
+                                { type: 'version', cssName: 'version', label: i18n.version, value: artifact.version },
+                                { type: 'createdDate', cssName: 'created-date', label: i18n.createdDate, value: artifact.createdDate },
+                                { type: 'publishedDate', cssName: 'publish-date', label: i18n.publishDate, value: artifact.publishDate },
+                                { type: 'type', cssName: 'type', label: i18n.type, value: i18n[artifact.type] },
+                                { type: 'locationName', cssName: 'location', label: i18n.location, value: artifact.locationName },
+                                { type: 'regionName', cssName: 'region', label: i18n.region, value: artifact.regionName }
+                            ])
+                        }
+                    </div>
+                </div>
+            </div>
+        );
+    }
+}
+
+export default DetailPageSidebar;
+    
```

Comment:
Rewind - @chrisdhanaraj seems to have a plan: https://github.com/carbon-design-system/carbon-addons-bluemix-layout-react/pull/1/files#r128085449